### PR TITLE
Quick fail for old PR builds

### PR DESCRIPTION
**Pull request by ogrisel**

### CODE REVIEW

**Comment by ogrisel** [comment]
It seems to work :)

File: README.md [hunk 1]
```diff
@@ -39,11 +39,12 @@ In particular:
...
-**Note**: when registering your github repo on appveyor ci, don't forget to
-activate the "Rolling builds" option to automatically cancel build on intermediate
-push events to avoid clogging the build queue with useless jobs.
-
-![Rolling builds](https://www.dropbox.com/s/7ug7ynx7pxycay2/Screenshot%202015-10-30%2014.53.47.png?dl=1)
+**Note**: it is possible to activate the "Rolling builds" option to
+automatically cancel build on intermediate push events to avoid clogging
+the build queue with useless jobs. However it can be problematic as it can
+cancel builds triggered by direct push to master outside of any PR. Instead
+it the appveyor.yml file of this repo uses a specific powershell snippet
```
Comment (JanSchulz) [inline README.md]: `Instead+,+- it- the appveyor.yml file`?

**Comment by ogrisel** [comment]
Thanks!

File: appveyor.yml [hunk 2]
```diff
@@ -78,6 +78,15 @@ environment:
...
 install:
+  # If there is a newer build queued for the same PR, cancel this one.
+  # The AppVeyor 'rollout builds' option is supposed to serve the same
+  # purpose but it is problematic because it tends to cancel builds pushed
+  # directly to master instead of just PR builds (or the converse).
+  # credits: JuliaLang developers.
+  - ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_BUILD_NUMBER -ne ((Invoke-RestMethod `
+        https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
+        Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
+        throw "There are newer queued builds for this pull request, failing early." }
```
Comment (jayvdb) [inline appveyor.yml]: can this line be indented at little so it is more clearly one branch of the if?

Comment (ogrisel) [inline appveyor.yml]: Ok I will open another PR.

Comment (ogrisel) [inline appveyor.yml]: @jayvdb I pushed the fix directly in master.

**Pull request merged**